### DISCUSSION TITLE
Positive binding energy from MMPBSA analysis

### DISCUSSION THREAD

**Discussion by julianaamorim** [Valdes-Tresanco-MS/gmx_MMPBSA#77] [Q&A] [answered]
Hi,
I simulated my complex of interest using Gromacs-2021.1 version in CHARMM36FF for 50ns. Due to the presence of numerous interactions (the binding energy should be at negative values). However, I am getting positive binding energy between the complex and I am not sure if the reason is perhaps some error in the scripts. Could someone please help me with this problem? Thank you...

mmpbsa.in
&general
sys_name="Prot-Lig-CHARMM",
startframe=3001, endframe=5000, verbose=4,
/
&pb
# radiopt=0 is recommended which means using radii from the prmtop file for both the PB calculation and for the NP
# calculation
istrng=0.15, fillratio=4.0, radiopt=0

gmx_MMPBSA.log
[gmx_MMPBSA.log](https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/files/7112147/gmx_MMPBSA.log)

Differences (Complex - Receptor - Ligand):
Energy Component            Average              Std. Dev.   Std. Err. of Mean
-------------------------------------------------------------------------------
BOND                         0.0000                0.0001              0.0000
ANGLE                       -0.0000                0.0000              0.0000
DIHED                        0.0000                0.0000              0.0000
UB                           0.0000                0.0001              0.0000
IMP                         -0.0000                0.0001              0.0000
CMAP                         0.0000                0.0000              0.0000
VDWAALS                    -50.9073                3.8308              0.0857
EEL                        -52.7357               18.0597              0.4038
1-4 VDW                     -0.0000                0.0000              0.0000
1-4 EEL                      0.0000                0.0000              0.0000
EPB                         95.5043               11.6533              0.2606
ENPOLAR                    -37.6925                1.5446              0.0345
EDISPER                     67.6972                1.7129              0.0383

DELTA G gas               -103.6430               17.3714              0.3884
DELTA G solv               125.5090               11.6618              0.2608

DELTA TOTAL                 21.8660               10.4987              0.2348

**Comment by Valdes-Tresanco-MS** [comment, marked as answer]
Please, check this [thread](https://groups.google.com/g/gmx_mmpbsa/c/T8NOkmQgq50/m/rewCLpauAwAJ) in our Google Group

**Comment by julianaamorim** [comment]
Thanks Valdes, it worked!

:)))
